citrination_client/client.py

```diff
@@ -239,6 +237,57 @@ def upload_file(self, file_path, data_set_id, root_path=None):
                                    "status": r.status_code}
                 return json.dumps(message)
 
+    def upload_file(self, file_path, dataset_id, root_path=None):
+        """
+        Upload file to Citrination.
+        :param file_path: File path to upload.
+        :param data_set_id: The dataset id to upload the file to.
+        :return: Response object or return code if the file was not uploaded.
+        """
+        return self.transfer(dataset_id, file_path)
+
+    def move_or_copy(self, dataset_id, move_or_copy, original_path, target_path, is_dir=False):
+        """
+        Move a file within the "filesystem" of a dataset on Citrination.
+        :param dataset_id: The ID of the dataset in which the move will occur
+        :param original_path: The path (to a file or to a directory) to move
```

Comment:
What are valid values for `move_or_copy`?  What happens if someone specifies a different value?